os_net_config/impl_nmstate.py

```diff
@@ -1501,9 +1501,15 @@ def add_sriov_pf(self, sriov_pf):
         if sriov_pf.promisc:
             data[Interface.ACCEPT_ALL_MAC_ADDRESSES] = True
         if sriov_pf.link_mode == 'switchdev':
+            logger.info(f'enabling switchdev for sriov pf: {sriov_pf.name}')
```

Comment:
Instead lets have the default value and an override when switchdev is enabled ?
data[Ethtool.CONFIG_SUBTREE] = {}
            data[Ethtool.CONFIG_SUBTREE][Ethtool.Feature.CONFIG_SUBTREE] = {
                'hw-tc-offload': False}